src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -224,6 +247,15 @@ def _convert_str_to_enum_entry(args, option, enum):
 def _add_input_args(parser):
     input_group = parser.add_argument_group("Input")
 
+    input_group.add_argument(
+        "--batch-size",
+        "-b",
+        type=int,
+        default=LlmInputs.DEFAULT_BATCH_SIZE,
+        required=False,
+        help=f"The batch size of the requests GenAI-Perf should send",
```

Comment:
can we mention here that is only works for embeddings today?

Reply:
Wonderful catch! I forgot to add this when I restricted the batch size arg. Added!

Reply:
thank you